docs/integrations/optuna.rst

```diff
@@ -22,19 +22,21 @@ Initialize Neptune and create an experiment
 .. code-block:: python3
 
     import neptune
-    neptune.init('jakub-czakon/blog-hpo')
-    neptune.create_experiment(name='optuna sweep')
 
-Create **NeptuneMonitor** callback
-----------------------------------
-Pass the experiment object as the first argument.
+    neptune.init(api_token='ANONYMOUS',
+                 project_qualified_name='shared/showroom')
 
-.. note:: To be able to log information after the ``.fit()`` method finishes, remember to pass ``close_after_train=False``.
+    neptune.create_experiment('optuna-sweep')
```

Comment:
in the app it is named: "optuna-integration"

Reply:
I know, but I don't want to re-do it